### PR TITLE
German translations

### PR DESCRIPTION
**Pull request by AlessandroFasse**
Added the translation for the Expansion "Menagerie" in German and also the missing "Kaptain Tobias". 

Also I added that you can run the tests on Windows. There was an issue with the path separator `\` on Windows, which was miss interpreted. 

I hope everything else is with respect to the guidelines.

### CODE REVIEW

File: card_db_src/de/sets_de.json [hunk 1]
```diff
@@ -101,10 +101,20 @@
...
...n! Von jedem kann man etwas lernen, etwa, wie man wirklich weit spucken kann oder welche Sorte Gras am besten schmeckt. Gar nicht so leicht, sich das alles zu merken, aber du bist wie ein Elefant: Du vergisst nichts. Und du hast Angst vor Mäusen. Kürzlich hast du mit dem Reiten angefangen. Pferde sind angsteinflößend; es heißt, man kann sie zur Tränke führen, aber das hast du noch nicht geschafft. Du arbeitest dich langsam nach oben und hast mit Hunden angefangen. Bisher läuft es gut: Der Hund hat dich noch nicht abgeworfen. Deine Menagerie ist noch übersichtlich, mit einer Ziege, zwei Ratten und dem Berater, der die Menagerie vorgeschlagen hat, Den Fuchs hast du nicht gekriegt, aber der war vermutlich sowieso böse, immerhin hast du jetzt ein paar Kamele, die sich allerdings genauso bockig anstellen wie erwartet, und eine Schildkröte, die den Atem länger anhalten kann, als es irgendjemanden interessiert. Das Königreich der Tiere wird bald deins sein. Dies ist die 13. <i>Dominion</i>-Erweiterung, Sie hat 400 Karten, darunter 300 neue Königreichkarten. Menagerie ist eine <i>Dominion</i>-Erweiterung und kann nur zusammen mit einem Basisspiel oder dem Basiskarten-Set gespielt werden.",
```
Comment (sumpfork) [inline card_db_src/de/sets_de.json:110]: Punkt fehlt nach them ersten Satz. Komma vor "..., Den Fuchs" sollte ein Punkt sein. "irgendjemanden" -> "irgend jemanden". Komma nach "Erweiterung" sollte ein Punkt sein.

File: card_db_src/de/sets_de.json [hunk 2]
```diff
@@ -118,7 +128,7 @@
     },
     "promo": {
         "set_name": "Promo",
-        "set_text": "Promotional cards are official <i>Dominion</i> kingdom cards and Events which do not belong to any particular set. They are typically either released to help promote <i>Dominion</i> at conventions (such as Origins or GenCon), or to commemorate the anniversary of another game in order to help promote both.",
+        "set_text": "Die Promotionskarten sind offizielle <i>Dominion</i> Königreichkarten und Events, die zu keiner Erweiterung gehören. In der Regel werden diese Karten wärend Events (Origins or GenCon) oder an Jubiläen herhausgegeben um <i>Dominion</i> weiter zu promoten. ",
```
Comment (sumpfork) [inline card_db_src/de/sets_de.json:131]: "wärend" -> "während"

**Review by sumpfork — CHANGES_REQUESTED** [review]
Thanks for all the work!

I just started looking at this - haven't actually run it yet to check the output, but there seem to be a couple minor spelling/punctuation problems. I didn't go through all the text, but maybe you could check the rest for similar issues?

File: card_db_src/de/sets_de.json (hunk 1 above)
Comment (AlessandroFasse) [reply card_db_src/de/sets_de.json]: Danke

File: card_db_src/de/sets_de.json (hunk 2 above)
Comment (AlessandroFasse) [reply card_db_src/de/sets_de.json]: fixed

**Comment by AlessandroFasse** [comment]
Is also went now with a type check over all cards and found a lot of other mistakes in other editions as well

**Comment by AlessandroFasse** [comment]
Fixed the typo

**Pull request merged**